pkg/promtail/targets/journaltarget.go

```diff
@@ -154,18 +154,8 @@ func journalTargetWithReader(
 		return nil, errors.Wrap(err, "parsing journal reader 'max_age' config value")
 	}
 
-	// Default to system path if not defined. Passing an empty string to
-	// sdjournal is valid but forces reads from the journal to be from
-	// the local machine id only, which contradicts the default behavior
-	// of when a path is specified. To standardize, we manually default the
-	// path here.
-	journalPath := targetConfig.Path
-	if journalPath == "" {
-		journalPath = "/var/log/journal"
-	}
```

Comment:
Removing some defaults here, but not adding the described new ones ( `/var/log/journal`
and `/run/log/journal`).

Where do these come from then?

Reply:
They're deep within the [systemd code](https://github.com/systemd/systemd/blob/28b05d1e6ece19cd83afaf8f12f3fc5418151a29/src/journal/sd-journal.c#L1747-L1766)